### PR TITLE
Concat longitude and latitude and add location url on Setup tab

### PR DESCRIPTION
**Pull request by HThuren**
This PR concat longitude and latitude and add location url on Setup tab.
Same url as in gps tab
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

File: src/tabs/setup.html
Comment (haslinghuis) [inline src/tabs/setup.html:132]: @HThuren please fix:

- In Setup tab `latLon` is not defined
- In GPS tab `gpsLat` and `gpsLon` are still being referenced.

```
i18next::translator: missingKey en messages gpsLatLon gpsLatLon
three.module.js:21954 THREE.WebGLRenderer 97
i18next.js:22 i18next::translator: missingKey en messages gpsLat gpsLat
i18next.js:22 i18next::translator: missingKey en messages gpsLon gpsLon
gps.js:314 Online
```

Comment (HThuren) [reply src/tabs/setup.html:132]: Right, this was part of #3486, but not to wait on dialog about that PRm I do another to fix the GPS tab. Same design as on SETUP tab, since the url are the same for both lat and lon.
image

**Pull request merged**
